website/www/site/data/meetings.yml

```diff
@@ -9,31 +9,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Welcome to Jekyll!
 
 events:
-- date: 2016/04/01
-  time: "9:30 - 16:00 Pacific"
-  location: PayPal<br>San Jose, CA, USA
-  type: Dev/PPMC Meeting
-  materials:
-    - title: Presentation - PPMC Deep Dive
-      link: "https://docs.google.com/presentation/d/1uTb7dx4-Y2OM_B0_3XF_whwAL2FlDTTuq2QzP9sJ4Mg/edit?usp=sharing"
+  - date: 2016/04/01
+    time: "9:30 - 16:00 Pacific"
+    location: PayPal<br>San Jose, CA, USA
+    type: Dev/PPMC Meeting
+    materials:
+      - title: Presentation - PPMC Deep Dive
+        link: "https://docs.google.com/presentation/d/1uTb7dx4-Y2OM_B0_3XF_whwAL2FlDTTuq2QzP9sJ4Mg/edit?usp=sharing"
 
-    - title: Notes - PPMC Deep Dive
-      link: "https://docs.google.com/document/d/1SXSLj7FMIgKqj43nTcczFpJzqASeUMUCpbyklk2fBkg/edit?usp=sharing"
-  notes:
+      - title: Notes - PPMC Deep Dive
+        link: "https://docs.google.com/document/d/1SXSLj7FMIgKqj43nTcczFpJzqASeUMUCpbyklk2fBkg/edit?usp=sharing"
+    notes:
 
-- date: 2016/05/04
-  time: "8:00 - 11:00 Pacific"
-  location: Virtual
-  type: Technical Deep Dive
-  materials:
-    - title: Presentation - Beam Community Meeting
-      link: "https://drive.google.com/open?id=17i7SHViboWtLEZw27iabdMisPl987WWxvapJaXg_dEE"
+  - date: 2016/05/04
+    time: "8:00 - 11:00 Pacific"
+    location: Virtual
+    type: Technical Deep Dive
+    materials:
+      - title: Presentation - Beam Community Meeting
+        link: "https://drive.google.com/open?id=17i7SHViboWtLEZw27iabdMisPl987WWxvapJaXg_dEE"
 
-    - title: Notes - Beam Community Meeting
-      link: "https://drive.google.com/open?id=1szhEE_pfhEtrQye61jXAidUcMW7oebZCRc2InUe3ou0"
-  notes:
+      - title: Notes - Beam Community Meeting
+        link: "https://drive.google.com/open?id=1szhEE_pfhEtrQye61jXAidUcMW7oebZCRc2InUe3ou0"
+    notes:
```

Comment:
Are the whitespace changes in these yaml files necessary?

Reply:
@TheNeuralBit: The file is formatted in the correct form according to the Indentation rule [1].

[1] https://docs.saltstack.com/en/master/topics/troubleshooting/yaml_idiosyncrasies.html